### DISCUSSION TITLE
Is there a way to set the Grafana link default via environment variable?

### DISCUSSION THREAD

**Discussion by junderw** [teslamate-org/teslamate#1893] [Q&A] [answered]
I am working on adding Teslamate support for a standalone RasPi device called Umbrel. [PR is here](https://github.com/getumbrel/umbrel/pull/891).

I think it would be better UX for users (who are looking for one-click) if I could put in a default value for the grafana link via ENV in the docker-compose... that way they can access Grafana from the web portal directly without needing to worry about it.

I was wondering if there's a way to do that currently... if not, I would like to propose a new Idea thread with it.

Thanks.

Slightly Related: https://github.com/adriankumpf/teslamate/discussions/1892

**Comment by adriankumpf** [comment, marked as answer]
Currently that is not supported. It will, however, configure itself the first time a user clicks a link on any link that points to the TeslaMate UI in Grafana (e.g. clicking on a geo-fence or address on the Drives dashboard).

**Reply by junderw** [reply]
Thanks, I added a new Idea post #2010
